python/llm/src/bigdl/llm/transformers/gguf/gguf.py

```diff
@@ -246,6 +246,24 @@ def __iter__(self):
                 tensor = self.convert_funcs[qtype](tensor, size, ndims, dims)
                 yield name, tensor
 
+    def load_while_process(self, process):
```

Comment:
passing a handle to ggufFileLoader is better than applying iter as for performance test